packages/@remirror/core/src/builtins/__tests__/commands-extension.spec.ts

```diff
@@ -30,3 +30,30 @@ test('can chain commands', () => {
 
   expect(editor.state.doc).toEqualRemirrorDocument(doc(p(bold(italic('my content')))));
 });
+
+test('clears range selection', () => {
+  const text = 'my content';
+
+  const editor = renderEditor([]);
+  const { doc, p } = editor.nodes;
+  const { commands } = editor;
+
+  editor.add(doc(p(`<start>${text}<end>`)));
+
+  // Pre-condition: range selection covers complete text
+  expect(editor.state.selection.to).toBe(editor.state.selection.from + text.length);
+
+  expect(commands.clearRangeSelection()).toBeTrue();
+
+  expect(editor.state.selection.to).toBe(editor.state.selection.from);
```

Comment:
Probably want to add another condition here to ensure that the cursor hasn't just jumped to a new position with an empty selection, in case of future refactoring.

```suggestion
  expect(editor.state.selection.to).toBe(editor.state.selection.from);
  expect(editor.state.doc).toEqualRemirrorDocument(doc(p('')));
```